react/features/base/ui/Tokens.js

```diff
@@ -288,7 +288,13 @@ export const colorMap = {
     support09: 'support09',
 
     // Used for avatars and raise hand badge
-    support07: 'support07'
+    support07: 'support07',
+
+    // Colors for avatar status badge
```

Comment:
You had a good idea adding the color here, but unfortunately, you can't. The tokens are part of the design system which our designer is in charge of. We can't add tokens without his approval, and the tokens must also be used in multiple places. So please remove the colors from here. Extract them in a constant and use that constant instead of the theme. Thanks